tests/unit/systems/fil/test_forest.py

```diff
@@ -96,6 +97,45 @@ def test_export(tmpdir):
     assert parsed_config.backend == "fil"
 
 
+def test_export_merlin_models(tmpdir):
```

Comment:
This test only runs in Jenkins for now, can be activated in CPU / Github actions workflow by adding merlin models install in setup